app/models/qing_group_divided.rb

```diff
@@ -0,0 +1,13 @@
+class QingGroupDivided
+  include Translatable
+
+  translates :group_name
+
+  attr_accessor :hidden, :children
+
+  def initialize(qingGroup)
```

Comment:
Sorry! These camelCases, semicolons and unnecessary () on methods is coming from my javascript side. Will pay more attention to this! Thanks!